src/input.ts

```diff
@@ -89,10 +85,6 @@ export function validateInput(input: Input): Input {
     throw new Error("keep must be an integer between 0 and 100 (inclusive)")
   }
 
-  if (isRestPackageType(input.type) && !input.user && !input.organization) {
```

Comment:
We now need to always validate that `type` is set, because the rest api requires it.

Reply:
Added validation for type parameter, and set as required int he action.yaml file